src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -1022,6 +1009,26 @@ protected void wrapUp(GHCommit[] page) {
         };
     }
 
+    /**
+     * This method can get the last commit instance
+     * @return the last commit instance
+     */
+    public GHCommit getLastCommit(){
+        List<GHCommit> commits = listCommits().asList();
```

Comment:
This will get all the commits in the repository just to find the last one.  For large repos, that could be hundreds of requests, just to get one commit.  

Are the commits actually returned in an unordered fashion?